scripts/docker-compose.dev.yaml

```diff
@@ -1,8 +1,13 @@
+version: "3.0"
+name: memos-dev
 services:
   db:
     image: mysql
     volumes:
-      - ./.air/mysql:/var/lib/mysql
+      - ./../.air/mysql:/var/lib/mysql
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
```

Comment:
Some env var related to setting the password is [required](https://github.com/docker/kitematic/issues/2175).